### DISCUSSION TITLE
Aircraft remodel

### DISCUSSION THREAD

**Discussion by kilocharlie1** [flybywiresim/aircraft#7394] [Q&A] [answered]
Is there a timetable yet on when the flight deck/aircraft remodel will arrive?

**Comment by Benjozork** [comment, marked as answer]
Due to the community-driven nature of our projects, we do not enforce delivery timelines on our volunteer developers. We value high-quality, polished releases over frequent and quickly-released ones, which means some features might take time to land in a publicly-available build. Rest assured that our developers and contributors are working hard on making every FlyByWire project as bug-free and feature-complete as possible, but that we are not able to provide release dates or estimates for a certain feature/fix.
